src/ase/cw/model/Server.java

```diff
@@ -10,22 +13,27 @@
  */
 public class Server implements OrderConsumer {
 
+    private enum status {FREE, BUSY, PAUSED}
```

Comment:
Just saying:
A enum follows in Java the same naming Convention as a class (Since enums are classes)
Therfore it should start with a Upper letter: Status